### ISSUE TITLE
Add an option to configure query timeout

### ISSUE THREAD

**Issue by ransagy** [mtxr/vscode-sqltools#238]
While working on some heavier queries lately, I've come across the 15 second query timeout limit quite often. It would be useful to be able to configure it or even disable the timeout (per connection?) to allow for bigger/heavier queries to go through. Obviously, This shouldn't be off by default.
Changing the `connectionTimeout` setting has no effect on this either, it seems.

**Comment by mtxr** [comment]
@ransagy which dialect are you using?

**Comment by ransagy** [comment]
I have to remember writing those :]
I'm on MSSQL against an Azure SQL DB.

**Comment by mtxr** [comment]
@ransagy I believe this is fixed now per #235. I've tested with requestTimeout and tweaking pool config and it worked as expected.

 Try to tweak your settings a bit with this options: https://www.npmjs.com/package/mssql#general-same-for-all-drivers

Remember this should be inside of `mssqlOptions`. This is how we pass those options:
https://github.com/mtxr/vscode-sqltools/blob/4de76e30fb109538f09f1d4665aa335da3ad609e/packages/core/dialect/mssql/index.ts#L20-L45

Keep me posted. Closing.
